### PR TITLE
Added Pay by Bank logo

### PR DESCRIPTION
**Pull request by burgaard**
## Why are you adding this icons?
I'm adding/updating this icon(s) because ..

#### Help us identify yourself
- [ ] I'm working/collaborating with the brand directly and they have provided the icons.
- [x] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I have attached a screenshot comparison with the example icon provided in guidelines
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

## Attach a screenshot of the icon along side the example Visa icon

Screenshot 2023-10-13 at 2 06 16 PM
Screenshot 2023-10-13 at 2 06 24 PM

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)

@Jacquesattaque 

Marlena Spidalieri

What's the expected date of this change to deploy on Shopify?

As soon as possible.

### CODE REVIEW

**Comment by Jacquesattaque** [comment]
@burgaard If the intention of this icon is to highlight a generic payment method, I'd recommend using our generic [online banking icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/onlinebanking.svg). We're working on integrating this with merchant branding in checkout as well so it will seamlessly adapt to any merchant checkout customizations.

**Comment by burgaard** [comment]
@Jacquesattaque this is not generic -- it's our Pay by Bank logo, font and color style consistent with the rest of our branding

**Comment by Jacquesattaque** [comment]
@burgaard In that case there is already a Pay By Bank payment method and icon. Are you trying to replace that one, or is this new? If it's new, you'll need to choose a new name for this payment method. Underscores are not permitted in the name.

**Comment by burgaard** [comment]
@Jacquesattaque it's our unique logo -- we are not trying to replace any of the existing logos. I'm confused about the underscore requirement -- there are lots of other entries with underscores in the name field?

**Comment by burgaard** [comment]
OK, I've updated the name @Jacquesattaque

**Review by Jacquesattaque — CHANGES_REQUESTED** [review]
Two changes to the icon:

- Looks like you have the border and background layers duplicated here. Remove one set.
- There are quite a few unnecessary points in drawing the letters, which is increasing file size. Try to simplify this to reduce file size.

Screenshot 2023-10-19 at 9 37 02 AM

**Comment by burgaard** [comment]
@Jacquesattaque I've fixed the duplicated borders. The vector points for the letters are simply just how the font is rendered in the SVG export.

**Review by Jacquesattaque — APPROVED** [review]

**Comment by burgaard** [comment]
@Jacquesattaque do you have an estimate when this might be available on Shopify?

**Comment by Jacquesattaque** [comment]
@burgaard We do a monthly release, typically on the first Wednesday of the month. The next release is tentatively scheduled for next Wednesday, November 1.

**Comment by adeniyiao** [comment]
Hi @burgaard 

Could you please rebase this branch with activemerchant/master to resolve the conflicts.

Thanks

**Comment by burgaard** [comment]
Done @adeniyiao!

**Comment by adeniyiao** [comment]
Thanks for quick response @burgaard , kindly fix the test failure below. 

image

File: db/payment_icons.yml [hunk 1]
```diff
@@ -575,8 +575,8 @@
   label: EFT Secure
   group: bank_transfers
 -
-  name: mpesa
   label: M-Pesa
+  name: mpesa
```
Comment (adeniyiao) [inline db/payment_icons.yml:579]: Unexpected change here as well

**Comment by burgaard** [comment]
@adeniyiao should be fixed now!

File: db/payment_icons.yml (hunk 1 above)
Comment (burgaard) [reply db/payment_icons.yml]: Not sure where that came from. Fixed!

**Pull request merged**